cmd/tsconnect/wasm/wasm_js.go

```diff
@@ -323,6 +343,120 @@ func (i *jsIPN) logout() {
 	go i.lb.Logout()
 }
 
+func (i *jsIPN) tcp(host string, port int, config js.Value) interface{} {
```

Comment:
we should probably expose the timeout in this signature.

also: document what "config" is.